static/usage/modal/can-dismiss/boolean/demo.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Modal | Can Dismiss</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+  <style>
+    ion-modal {
+      background: rgba(0, 0, 0, 0.2);
+    }
+  </style>
+
+</head>
+
+<body>
+  <ion-app>
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>App</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content class="ion-padding">
+      <ion-button id="open-modal" expand="block">Open</ion-button>
+
+      <ion-modal trigger="open-modal" swipe-to-close="true">
```

Comment:
Swipe to close was deprecated, so I don't think we should be showing new devs how to use it.

Reply:
Also the card modal looks weird: 
![image](https://user-images.githubusercontent.com/2721089/167484064-82323dd6-d0f0-4fac-a571-79a9bed5e79f.png)

You probably need to wrap the header/content in `ion-app` in `.ion-page`.